Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Bindings-Forms.swift

```diff
@@ -56,20 +56,20 @@ struct BindingFormView: View {
           HStack {
             TextField(
               "Type here",
-              text: viewStore.binding(keyPath: \.text, send: BindingFormAction.binding)
+              text: viewStore.binding(BindingFormAction.binding).text
```

Comment:
It's decently shorter, so might be worth it. It also reads a bit jumbly right now: a lot of `binding.binding` noise. I wonder if we can come up with a less repetitive call site.

Reply:
yeah I like this!